README.md

```diff
@@ -1,208 +1,262 @@
 
-### Step 2 - Create APP ID
+> Note: don't forget to set the `Project security` -> `User roles` to include `PushNotifications.Administrator` role as part as the main `Administrator` role and `PushNotifications.User` role as part of the main `User` role.
+>
+> <img src="assets/images/implementation guide/Project security.JPG"/>
 
-Click on Identifiers and then click on App IDs.
-On the top right there will be a plus button, press this and a dialog like this should appear:
-<img src="assets/images/apn-step2.PNG"/>
+At this moment you can deploy your application to the cloud. If you are using Free App, simply click the `Run` button.
 
-Enter your app ID name and select Explicit App ID. You must select Explicit App ID inorder to be able to perform push notifications.
-Enter a Bundle ID, this bundle ID must match the bundle ID that you entered during the phonegap build phase.
-<img src="assets/images/apn-step2-1.PNG"/>
-Tick the options push notifications and then click continue.
+### Step 8 - Set up access to APNs and GCM
 
-### Step 3 - Confirm
-Click submit
-<img src="assets/images/apn-step3.PNG"/>
+Set up access to APNs and GCM and configure them in your application.
 
-### Step 4 - Creating APS Certificate
-Locate your created App ID and click on it. This will expand it out.
-<img src="assets/images/apn-step4.PNG"/>
+See [Setting up Apple Push Notification Server](#setting-up-apple-push-notification-server) and [Setting up Google Cloud Messaging Server](#setting-up-google-cloud-messaging-server) for the details.
 
-Click on Edit and scroll down to the push notifications section. You will see that there are two options one to generate a certificate for development purposes and one for production. For the purpose of this documentation we will generate a development certificate, so we will click on the development create certificate button.
-<img src="assets/images/apn-step4-1.PNG"/>
+### Step 9 - Build the hybrid mobile application
 
-### Step 5 - Generating Certificate 
-Click continue
-<img src="assets/images/apn-step5.PNG"/>
+You will need to build the hybrid mobile application. Refer to the [Publishing a Mendix Hybrid Mobile App howto] (https://world.mendix.com/display/howto50/Publishing+a+Mendix+Hybrid+Mobile+App+in+Mobile+App+Stores) to get the explanation on how to do that. Note that you should opt to download the app instead of directly publishing it. This is necessary because we need to include a phonegap plugin which is used by this module into the mobile hybrid application.
 
-You will then be asked to upload a certificate signing request file. In order to create this file please read the following documentation:
+Once you have the mobile hybrid project file downloaded, extract it and include the required phonegap plugin by adding this line to the `config.xml` file:
```

Comment:
hybrid mobile

'Once you have downloaded the hybrid mobile project file'

PhoneGap

Technically it might be plug-in, but seen both versions used extensively

'the following line' instead of 'this line'?